src/components/Header/NavDrawer.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { CloseIcon, HamburgerIcon } from '@chakra-ui/icons';
+import {
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  Flex,
+  IconButton,
+  useColorModeValue,
+  useDisclosure
+} from '@chakra-ui/react';
+
```

Comment:
any specific reason for having extra space?

Reply:
It's from eslint: "simple-import-sort/imports"
<img width="475" alt="eslint" src="https://user-images.githubusercontent.com/80061297/182048693-ea9552a3-a888-4d4e-9fe2-a98561ad57b3.PNG">